nominal/core/client.py

```diff
@@ -78,6 +79,52 @@
 class NominalClient:
     _clients: ClientsBunch = field(repr=False)
 
+    @classmethod
+    def from_profile(
+        cls,
+        profile: str,
+        *,
+        trust_store_path: str | None = None,
+        connect_timeout: timedelta = timedelta(seconds=30),
+    ) -> Self:
+        """Create a connection to the Nominal platform from a named profile in the Nominal config.
+
+        Args:
+            profile: profile name in the Nominal config.
+            trust_store_path: path to a trust store certificate chain to initiate SSL connections. If not provided,
+                certifi's trust store is used.
+            connect_timeout: Request connection timeout.
+        """
+        config = NominalConfig.from_yaml()
```

Comment:
Just to note: it says in several places the config is stored at ~/.config/nominal "by default", but really the assumption is implicit in statements like these.